docs/html/reference/pip_install.rst

```diff
@@ -1000,6 +1000,11 @@ Examples
 
          py -m pip install --upgrade SomePackage
 
+    .. note::
+
+      This will only update ``SomePackage`` as it is a direct requirement. Any
```

Comment:
From @uranusjr from prior PR:

> Indirect requirements can still be upgraded here if a later version of `SomePackage` needs them upgraded. Say
> 
>     * `SomePackage==1.0` needs `DepPackage==1.0`
> 
>     * `SomePackage==2.0` needs `DepPackage==2.0`
> 
> 
> With `SomePackage==1.0` and `DepPackage==1.0` initially installed, `pip install --upgrade SomePackage` will still upgrade `DepPackage`.
>
> IOW, only `SomePackage` is _guarenteed_ to be upgraded. Its dependencies are upgraded only if needed.

Reply:
Done in a93ec25a6, I think.

I guess I'm still a bit confused as to the difference, then, between `only-if-needed` vs. `to-satisfy-only`, as they seem to do the same thing? (I'm having trouble of finding a different behavior...)

Reply:
Ah, I think I get it now; if `SomePackage` is already installed, but is older than latest:

* `pip install --update --update-strategy=eager SomePackage` updates `SomePackage` and its dependencies to latest (compatible?) versions
* `pip install --update --update-strategy=only-if-needed SomePackage` updates `SomePackage` to latest, and only its deps if necessary.
* `pip install --update --update-strategy=to-satisfy-only SomePackage` will not update `SomePackage` nor its deps

Is that right?

Reply:
K, yeah, this matches what you said here:
https://github.com/pypa/pip/issues/8726#issuecomment-671073925

Though I will try to reword your explanation here.

Reply:
Reworded in 9d7ca273d